pkg/logger/logger_test.go

```diff
@@ -189,6 +189,51 @@ func TestDebug(t *testing.T) {
 	}
 }
 
+func TestLogLevelFromEnv(t *testing.T) {
+	mu.Lock()
+	defer mu.Unlock()
+	flag.Set(FlagLevelName, seelog.DebugStr)
+	clean()
+	os.Setenv("ALIYUN_LOGTAIL_LOG_LEVEL", "info")
```

Comment:
info is the default level, maybe we should test debug level to see if the env works?